samples/apps/copilot-chat-app/webapp/tests/chat.test.ts

```diff
@@ -1,51 +1,50 @@
-import { expect, test } from '@playwright/test';
+/* eslint-disable testing-library/prefer-screen-queries */
+import { test } from '@playwright/test';
+import * as util from './utils'
+import * as simpletests from './testsBasic'
+import * as plannertests from './testsPlanner'
+import { multiUserTest } from './testsMultiuser'
 
-/*
-This test is a simple end-to-end test that verifies the app can be
-loaded, and signed in. The user can send a message to the bot, and expect
-a response.
-*/
-test('get response from bot', async ({ page }) => {
-    // Make sure the server is running.
-    await page.goto('https://localhost:40443/healthz');
-    expect(page.getByText('Healthy')).toBeDefined();
+test.describe('Copilot Chat App Test Suite', () => {
```

Comment:
nit: I think we could shorten this to:
```suggestion
A new chat session is opened for each test so that the chat history is not polluted and the LLM is not confused.
```

Reply:
The change you're suggesting makes the comment one big line instead of 2 short ones; I think the multiple line approach is more readable (no scrolling or needing to have word wrap enabled in the IDE). We also follow this approach else where in the code.